site/components/pages/Infinity/index.tsx

```diff
@@ -70,7 +71,7 @@ export const Infinity: NextPage<Props> = () => {
       />
 
       <Navigation activePage="Infinity" showThemeToggle={false} />
-
+      <GetStardedModal showModal={showModal} setShowModal={setShowModal} />
```

Comment:
We might need to render this element immediately after the button so that when the user presses `tab` the focus is able to reach the modal.

```
<button>
<modal />
```

Reply:
rendering the modal after the button isn't making the tab focus on the modal. do you think we will need another focus trap here @Atmosfearful ?

Reply:
My suggestion is to pull this out and fix it after I create a reusable `useFocus` hook in #684 because the modal is rendered under the nav if it is rendered after the button.